Tests/Command/GenerateAdminCommandTest.php

```diff
@@ -125,14 +123,14 @@ public function testExecute()
         $command = $this->application->find('sonata:admin:generate');
         $commandTester = new CommandTester($command);
         $commandTester->execute(array(
-            'command' => $command->getName(),
-            'model'=>'Sonata\AdminBundle\Tests\Fixtures\Bundle\Entity\Foo',
-            '--bundle'=>'AcmeDemoBundle',
-            '--admin'=>'FooAdmin',
-            '--controller'=>'FooAdminController',
-            '--services'=>'admin.yml',
-            '--id'=>'acme_demo_admin.admin.foo',
-            ), array('interactive'=>false,));
+            'command'      => $command->getName(),
+            'model'        => 'Sonata\AdminBundle\Tests\Fixtures\Bundle\Entity\Foo',
+            '--bundle'     => 'AcmeDemoBundle',
+            '--admin'      => 'FooAdmin',
+            '--controller' => 'FooAdminController',
+            '--services'   => 'admin.yml',
+            '--id'         => 'acme_demo_admin.admin.foo',
+            ), array('interactive' => false));
```

Comment:
This is an auto-fix by php-cs-fixer. If I move tab, it will be redone like that.

Or I deactivate this fixer completely.

What is your choice?